routes/user.js

```diff
@@ -161,4 +161,39 @@ router.put("/updateStatus", async function (req, res) {
     });
 });
 
+router.get("/search/:userName", async function (req, res) {
+    if (!req.headers.authorization) {
+        return res.status(400).json({
+            message: "Invalid request",
+        });
+    }
+
+    const userID = verifyToken(req.headers.authorization);
+
+    if (!userID) {
+        return res.status(440).json({
+            message: "Invalid Credentials",
+        });
+    }
+        
+    if (!req.body.status) {
```

Comment:
Could we just put this with the top if 400 case?

Reply:
fixed it mb lol